src/Aspire.Cli/Commands/DeployCommand.cs

```diff
@@ -19,17 +18,15 @@ public DeployCommand(IDotNetCliRunner runner, IInteractionService interactionSer
     {
     }
 
+    protected override string OperationPrefix => "Deployment";
```

Comment:
```suggestion
    protected override string OperationSucceededPrefix => DeploymentStrings.OperationSucceededPrefix;
    protected override string OperationFailedPrefix => DeploymentStrings.OperationFailedPrefix;
```

Add localizations of these values.